api/v1alpha3/common_types.go

```diff
@@ -123,50 +122,6 @@ type MachineAddresses []MachineAddress
 // In future versions, controller-tools@v2 might allow overriding the type and validation for embedded
 // types. When that happens, this hack should be revisited.
 type ObjectMeta struct {
```

Comment:
I don't think we can and should remove anything in our v1alpha3/v1alpha4 packages. It's a breaking change if we remove fields of our "frozen" old API versions